cogs/study_groups.py

```diff
@@ -134,6 +134,43 @@ async def _group(self, ctx: Context):
         member: Union[Member, User] = ctx.author
         logger.info(f'User="{member.name}#{member.discriminator}({member.id})", Command="{ctx.message.content}"')
 
+    @_group.command(pass_context=True,
+                    name="create",
+                    brief="Creates a Study group.",
+                    help="The name must contain the tag and the semester number.")
+    async def group_create(self, ctx: Context, name: str, semester: int = 1, color: int = 0):
+        """
+        Adds a new role-channel pair as a group.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            name: The name of the role and the chat.
+
+            color: color of group.
+
+            semester: number of semester
+        """
+        color_db = PrimitiveMongoData(CollectionEnum.GROUP_COLOR)
+        active_db = PrimitiveMongoData(CollectionEnum.GROUP_ACTIVE)
+        guild: Guild = ctx.guild
+
+        await color_db.insert_one({name: color})
+        await active_db.insert_one({name: True})
```

Comment:
check if there isn't already a group with that name

Reply:
Will be checked new